logs/logs.go

```diff
@@ -0,0 +1,53 @@
+// Package logs provides the standard interface and handler for OpenFaaS providers to expose function logs.
+//
+package logs
+
+import (
+	"fmt"
+	"time"
+)
+
+// Request is the query to return the function logs.
+type Request struct {
+	// Name is the function name and is required
+	Name string `json:"name"`
+	// Instance is the optional container name, that allows you to request logs from a specific function instance
+	Instance string `json:"instance"`
+	// Since is the optional datetime value to start the logs from
+	Since *time.Time `json:"since"`
+	// Limit sets the maximum number of log messages to return, <=0 means unlimited
+	Limit int `json:"limit"`
+	// Follow is allows the user to request a stream of logs
+	Follow bool `json:"follow"`
+	// Pattern is an optional regexp value to filter the log messages
+	Pattern *string `json:"pattern"`
+	// Invert allows you to control if the Pattern should be matched or negated
```

Comment:
If that's the case wouldn't `InvertPattern` make more sense to first-time-readers?

Reply:
This is renamed